usaspending_api/etl/es_etl_helpers.py

```diff
@@ -150,6 +166,15 @@ def __init__(self, *args):
 # ==============================================================================
 
 
+def convert_postgres_array_as_string_to_list(array_as_string: str) -> list:
+    """
+        Postgres arrays are stored in CSVs as strings. Elasticsearch is able to handle lists of items, but needs to
+        be passed a list instead of a string. In the case of an empty array, return null
+        For example, "{this,is,a,postgres,array}" -> ["this", "is", "a", "postgres", "array"].
+    """
+    return array_as_string.replace("{", "").replace("}", "").split(",") if len(array_as_string) > 2 else None
```

Comment:
This has a risk of replacing things inside the array (possibly escaped). A better strategy would be to trim the first and last character.

Reply:
This doesn't help with nested arrays, but I think we can all agree that's out of scope.

Reply:
I agree with both comments. This will only work for something that is a simple Postgres array. Will make the change suggested in the first comment.